src/dma/collector/workflows/readiness_check/_postgres.py

```diff
@@ -11,6 +12,44 @@
 
     from dma.collector.query_managers import CanonicalQueryManager
 
+dbTypeMap = {
```

Comment:
nit: can we make variables `snake_case`?
```suggestion
db_type_map = {
```